src/sql/meson.build

```diff
@@ -0,0 +1,90 @@
+sql_files = files(
```

Comment:
It'd be nice if we could glob this but it looks like that's not how it's supposed to be done in meson land

Reply:
Someday Meson will have a Nix backend using dynamic derivations, and then this will work :)